app/scripts/app.js

```diff
@@ -47,7 +47,8 @@ app.config(function($locationProvider, $stateProvider, $urlRouterProvider, confi
                     var filters = {
                         keyword: $stateParams.keyword,
                         organization: $stateParams.organization,
-                        state: $stateParams.state
+                        state: $stateParams.state,
+                        status: $stateParams.status
```

Comment:
Meh… For an end-user, “result” would be better. For devs handling the code too, consistency is better. Since we're currently both and this is a power-user feature as long as there's no GUI (and the name itself won't be important if there is one), let's be lazy and not change what you implemented.